src/platform/editor/monaco.service.ts

```diff
@@ -10,14 +11,15 @@ export class MonacoService {
 
   private baseHref: string;
 
-  constructor(@Inject(NGX_MONACO_EDITOR_CONFIG) private config: NgxMonacoEditorConfig) {
+  constructor(@Inject(NGX_MONACO_EDITOR_CONFIG) private config: NgxMonacoEditorConfig, platformLocation: PlatformLocation) {
+    this.baseHref = platformLocation.getBaseHrefFromDOM();
   }
 
   loadMonaco(): Promise<void> {
     if (!loadedMonaco) {
       loadedMonaco = true;
       loadPromise = new Promise<void>((resolve: any) => {
-        const baseUrl = this.config.baseUrl || '/assets';
+        const baseUrl = this.config.baseUrl || Location.joinWithSlash(this.baseHref, '/assets');
```

Comment:
Isn't it enough to replace `/assets` with `./assets`? I got it running on github pages this way.